data/jobs.txt

```diff
@@ -1602,106 +1737,917 @@ mission "Bulk Delivery [1]"
 		distance 3 12
 		government "Republic" "Free Worlds" "Syndicate" "Neutral"
 	on complete
-		payment
-		payment 2000
-		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+		payment
+		payment 2000
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Bulk Delivery [2]"
+	name "Bulk delivery to <planet>"
+	job
+	repeat
+	description "Deliver <cargo> to <destination>. Payment is <payment>."
+	cargo random 25 2 .05
+	to offer
+		random < 50
+	source
+		attributes mining textiles factory farming fishing oil
```

Comment:
I am not sure upon the original intention, and I did not touch the Bulk Delivery [2] mission. I only moved the entire Bulk delivery block.
But I assume that the attributes line there is intentional.
This way a factory world can offer more jobs than a vanilla world. Vanilla can offer Bulk [0]+[1], factory can offer [0]+[1]+[2] at the same time.